inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/InlongClient.java

```diff
@@ -99,10 +99,11 @@ static InlongClient create(String serviceUrl, ClientConfiguration configuration)
      * List group status
      *
      * @param groupIds inlong group id list
+     * @param credentials auth info to query sort task such as sort cluster token
      * @return map of inlong group status list
      * @throws Exception the exception
      */
-    Map<String, InlongGroupStatusInfo> listGroupStatus(List<String> groupIds) throws Exception;
+    Map<String, InlongGroupStatusInfo> listGroupStatus(List<String> groupIds, String credentials) throws Exception;
```

Comment:
I suggest reserve `listGroupStatus(List<String> groupIds)`  and new add `listGroupStatus(List<String> groupIds, String credentials)`

Reply:
Agreed